src/Features/CSharp/Portable/GenerateMember/GenerateVariable/CSharpGenerateVariableService.cs

```diff
@@ -30,10 +29,12 @@ public CSharpGenerateVariableService()
         }
 
         protected override bool IsExplicitInterfaceGeneration(SyntaxNode node)
-            => node is PropertyDeclarationSyntax;
+            => node.IsKind(SyntaxKind.PropertyDeclaration);
```

Comment:
Why was this change made?

Reply:
It should be faster to just check enum value than try cast one type to another. At the same time I don't see readability regression here. So consider it as a simple refactoring